Pull request dysonfrost

dysonfrost opened the pull request:
Hello, 

I wanted to add a feature to adjust the position and the size of the player and the browser.
You can find an example of the result [here](https://imgur.com/BLIwcUp)

Basically, I'm using xdotool to get a Window ID of the player/browser then I adjust the position of the windows.

dysonfrost committed:
HTML capabilities and Twitch chat URL update

dysonfrost committed:
HTML capabilities + twitch chat URL update

Rofi couldn't escape characters like ampersands, added an HTML encode module using perl.
Updated the twitch chat bloc with new URL, making sure the web page loaded doesn't contain any blank space.

indeedwatson committed:
Merge pull request #5 from dysonfrost/master

HTML capabilities + twitch chat URL update

dysonfrost committed:
Add feature to adjust window size & position

Add xdotool dependency

dysonfrost committed:
Resolve merge conflict

dysonfrost committed:
Bracket fix

dysonfrost committed:
Add chrome/chromium compatibility

dysonfrost committed:
Add chrome/chromium compatibility

dysonfrost commented:
Just added a conditional statement to adjust browser parameters wether you're using chrome/chromium or firefox.

indeedwatson reviewed (commented):
Hey I think the player should go in the settings file, and I'd default to mpv rather than vlc. I'm gonna implement that and a better way to enable the chat.

However the adjust window function seems to do nothing for me, not sure if it's cause I use i3 or I'm doing something wrong when testing it.

Could you explain the reason for changing /bin/bash to /usr/bin/env/bash?

dysonfrost committed:
mpv compatibility + custom player title bar

dysonfrost committed:
Update player compatibility + settings + README

dysonfrost committed:
Resolve merge conflict #2

dysonfrost commented:
Hey thanks for your feedback, I've updated the settings file and changed some parameters for streamlink to be compatible with either mpv or vlc. I also [tested it with i3](https://i.imgur.com/aO53Pkm.png), it should work like a charm. 

Also I updated the list of dependencies in README.md and added a block to configure twitchnotifier.cfg.

I reverted the change from `/usr/bin/env bash` to `/bin/bash`, usually I change the shebang to `/usr/bin/env` for portability reasons but it's not necessary with bash.


i didn't notice you pushed on master, I'm going to compare your changes to resolve conflicts.
